src/mw/cuda_exec.cpp

```diff
@@ -23,6 +23,24 @@
 
 #include "cpp_compile.hpp"
 
+#include <stb_image.h>
+#define KHRONOS_STATIC
+#include <ktx.h>
```

Comment:
Why are we including KTX in cuda_exec.cpp?

Reply:
this is resolved. now we have a new scheme that has been pushed for handling ktx